src/axom/inlet/LuaReader.hpp

```diff
@@ -15,6 +15,7 @@
 #define INLET_LUAMAP_HPP
 
 #include "axom/inlet/Reader.hpp"
+#include <sol/sol.hpp>
```

Comment:
Is there a missing edit to `src/axom/inlet/CMakeLists.txt` that adds `sol` as a dependency to `inlet`?
(I realize it is header only, but it should still be added).